package.json

```diff
@@ -166,6 +166,8 @@
   },
   "devDependencies": {
     "@ipld/garbage": "^6.0.0",
-    "aegir": "^45.0.1"
+    "aegir": "^45.0.7",
+    "eslint-config-ipfs": "^7.0.6",
+    "eslint-plugin-n": "^17.14.0"
```

Comment:
aegir should be contributing what it needs for linting, no? does it fail without these?

Reply:
it does fails without on my side.

Reply:
try removing these and push again, this changeset works for me locally without them and pushing a new commit might wake CI up, something odd going in GitHub Actions even after a re-start attempt

Reply:
```
$ npm ls --all | grep 'eslint-config-ipfs\|eslint-plugin-n'
│ │ ├─┬ eslint-config-ipfs@7.0.6
│ │ │ ├─┬ eslint-plugin-n@16.6.2
│ │ │ ├── eslint-plugin-n@16.6.2 deduped
│ │ ├── eslint-plugin-no-only-tests@3.3.0
│ │ ├── eslint-plugin-node@11.1.0 deduped
│ ├─┬ eslint-plugin-node@11.1.0
```

this is from a freshly installed tree without them, so they're there already from aegir.